### ISSUE TITLE
Why use this instead of collections plugin?

### ISSUE THREAD

**Issue by ghost** [jocelynlecomte/metalsmith-series#1]
See https://github.com/segmentio/metalsmith-collections

**Comment by jocelynlecomte** [comment]
You're right that most of the features overlap. Nevertheless, after reading the collections plugin docs, I think (but I may be wrong, because I didn't tried it) that some features are different:
1. The way the collections are stored in the metadata, which is useful when you want to get all the collections:
   - The collections plugin will store them in the global Metalsmith `metadata`, at the **root level**.
   - The series plugin will store them **under the series key**. In my opinion, this may make it easier to list all the series.
2. The way the collections are linked to their files, which is useful when you want to show the collection a file belongs to:
   - The collections plugin doesn't seem to link a file to its collection in the case you use a pattern. But this link indeed exists if you use the YAML `collection` metadata.
   - The series plugin always create a bidirectional association. The file knows the name and all the other attributes of the serie it belongs to.

Those were some quick thoughts on that matter. Let me know your ideas !
